sys/mips/timer.c

```diff
@@ -65,10 +80,10 @@ static int mips_timer_start(timer_t *tm, unsigned flags, const bintime_t start,
 
   device_t *dev = tm->tm_priv;
   mips_timer_state_t *state = dev->state;
-
+  state->sec = 0;
+  state->cntr_modulo = 0;
   state->period_cntr = bintime_mul(period, tm->tm_frequency).sec;
   state->compare.val = read_count(state);
-  state->last_count_lo = state->count.lo;
```

Comment:
Why did you remove this line? Where is `last_count_lo` initialized?

Reply:
I think you should explicitly set it to 0 before calling `read_count()`, since you initialize other fields to 0 explicitly too.

Reply:
You are right, it was a dumb decision, which can cause some confusion, thanks!